### DISCUSSION TITLE
Local setup works, but not on production

### DISCUSSION THREAD

**Discussion by funkhaus-wout** [umami-software/umami#2956] [General]
Hi,

We were able to setup Umami on our local environment.
Now we want to deploy on our production server, but we get Forbidden errors…
We simply uploaded the builded code, changed DB-credentials and uploaded the database.

What are we doing wrong?
Problem is we're on a shared hosting with node version 14, so we cannot build the project on production servers…
Please help!

**Comment by builtbybengt** [comment]
Are you using the same domain that you specified in your website settings?

**Comment by funkhaus-wout** [comment]
Hello,

Yes, we do.
The problem is that our hosting does not support Node, so we need to fix
that first I suppose.
Thanks for the reply. Have a great day!

Nog een erg fijne middag gewenst.

Met vriendelijke groeten,
Wout Verhoeven
Zaakvoerder Funkhaus

[image: funkhaus-logo] <https://www.funkhaus.be/>

Kantoor: Antwerpsesteenweg 338 A - 2390 Westmalle

+32 476 50 83 74 <+32476508374>
***@***.***
www.funkhaus.be

[image: button-appointment]
<https://www.funkhaus.be/boek-een-afspraak/>   [image:
button-review] <https://www.funkhaus.be/schrijf-een-review/>


*Tevreden over onze aanpak? Beveel ons aan in je netwerk, of schrijf een
review <https://www.funkhaus.be/schrijf-een-review/>.**Lees ook
onze interessante blog <https://www.funkhaus.be/blog/> met tips 'n tricks
over het web, digital trends, en online marketing.*


Op zo 22 sep 2024 om 21:10 schreef Bengt Mayr ***@***.***>:
